client/demo/bundles/hVNetworkOperator/config.json

```diff
@@ -26,6 +26,25 @@
             "showAcknowledgmentFooter": "OnlyForEmittingEntity",
             "type": "INPROGRESS",
             "consideredAcknowledgedForUserWhen": "AllEntitiesOfUserHaveAcknowledged"
+        },
+        "messagesAndQuestions": {
+            "name": "Messages and questions",
+            "description": "Messages and questions",
+            "userCard": {
+                "template": "usercard_messages_and_questions",
+                "startDateVisible": false,
+                "endDateVisible": false,
+                "recipientForInformationVisible" : true,
+                "severityVisible": false
+            },
+            "response": {
+                "state": "messagesAndQuestions",
+                "emittingEntityAllowedToRespond": "false"
+            },
+            "templateName": "messages_and_questions",
```

Comment:
Potentially rename to message or question instead of messages and questions to be consistent with core repo

Reply:
We already have message or question in another bundle , i do not think it is necessary to be consistent here with the core